clearwater_config_manager/scripts/upload_shared_ifcs_xml

```diff
@@ -52,13 +53,17 @@ rc=$?
 
 # Check the return code and log if appropriate.
 if [ $rc != 0 ] ; then
-  echo Upload shared IFCs configuration failed to $keypath  >&2
+  echo Upload shared iFC sets configuration failed to $keypath  >&2
   cat /tmp/upload-shared-ifcs-xml.stderr.$$                 >&2
   cat /tmp/upload-shared-ifcs-xml.stdout.$$                 >&2
+else
+  if [ $request_shared_ifcs != "Y" ]; then
+    echo "Uploading shared iFC sets, but shared iFC sets aren't being requested from the HSS by this deployment. To enable shared iFC sets, set request_shared_ifcs=Y in /etc/clearwater/shared_config, and follow the documented process to modify Clearwater configuration"
```

Comment:
I think we need to say somewhere in here like "unless you have been instructed not to".
(So that people aren't afraid to follow the non back-compatible changes instructions.)